src/lib/dependencies/index.js

```diff
@@ -35,7 +35,7 @@ function dependenciesStats (file) {
     return nuget.dependenciesStats(file);
   }
   if (file.type === 'dep') {
-    return nuget.dependenciesStats(file);
```

Comment:
Still not there ;-) Should be `file.text` not `file`.

Reply:
you're right, fixed